category/domain/category.go

```diff
@@ -37,6 +42,14 @@ type (
 		LinkTarget string
 		Media      Medias
 	}
+
+	// AdditionalAttributes - concrete additional category attributes
+	AdditionalAttributes struct {
+		Title            string
+		MarketingTitle   string
```

Comment:
The attributes need to be explained - what is the difference between MarketingTitle / Title..